### ISSUE TITLE
Class cast exception during conversion from ODT to PDF

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#203]
```
Converting the ODT in the source code zip below throws a class cast exception 
in itext 2.1.7 (com.lowagie.text.Phrase).

I think it is the call to the add(int, Object) method casting Object to Element.

```

Original issue reported on code.google.com by `medge1...@gmail.com` on 10 Dec 2012 at 12:43

Attachments:
- [odttopdf.zip](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-203/comment-0/odttopdf.zip)

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Thank a lot to have attached your odt which causes problem.

@Leszek : could you add this ODT in our JUnit with Isuse203.odt please. If you 
have no time, I will try to do it when I will have time.

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 10 Dec 2012 at 8:44
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Exception during conversion is fixed in 1.0.3 version
However the resulting pdf loses content of the original odt (graphic frames and 
tab stops are not supported). See 
https://code.google.com/p/xdocreport/wiki/ODFDOMConverterPDFViaIText to read 
about converter limitations 
```

Original comment by `abe...@gmail.com` on 20 Aug 2013 at 1:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `abe...@gmail.com` on 21 Aug 2013 at 10:37
- Changed state: **Fixed**
